src/global/global_init.cc

```diff
@@ -352,7 +363,7 @@ global_init(const std::map<std::string,std::string> *defaults,
   if (g_conf()->run_dir.length() &&
       code_env == CODE_ENVIRONMENT_DAEMON &&
       !(flags & CINIT_FLAG_NO_DAEMON_ACTIONS)) {
-    int r = ::mkdir(g_conf()->run_dir.c_str(), 0755);
+    int r = rpl_mkdir(g_conf()->run_dir.c_str(), 0755);
```

Comment:
`rpl_` <> `ceph_`

Reply:
``ceph_mkdir`` is already taken.

Reply:
Then still perhaps use something different from `rpl`?
how about `compat`
Which is perhaps even a better prefix for all functions that try to unify code between the different OSes.

Reply:
``compat`` sounds good, I'll go with that.